packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/datacentres/Datacentres.page.tsx

```diff
@@ -80,6 +89,28 @@ const DatagridCommercialRange = (vcdDatacentre: VCDDatacentre) => (
   </DataGridTextCell>
 );
 
+const DatagridVrackCell = (vcdDatacentre: VCDDatacentre) => {
+  const { data: url } = useNavigationGetUrl([
+    DEDICATED_PATH,
+    `${VRACK_PATH}/${vcdDatacentre.currentState?.vRack}`,
+    {},
+  ]);
+
+  return (
+    <DataGridTextCell>
+      {vcdDatacentre.currentState?.vRack ? (
+        <Links
+          href={url as string}
+          target="_blank"
+          type={LinkType.external}
+          label={vcdDatacentre.currentState?.vRack}
```

Comment:
No need for `?` as you check for undefined just before.
It is a bit of a nitpick but it shows that you won't get a link with no label, and as it could be something that you want to avoid, it would show a typescript error if someone removes later while refactoring the code